src/view/kc-webcam.js

```diff
@@ -3,15 +3,9 @@ import { LitElement, html } from 'lit-element';
 import { store } from '../model/store.js'
 
 export class KCWebCam extends connect(store)(LitElement) {
```

Comment:
There's a bit too much going on in this line for my liking...

Reply:
Suggestion, please.

Reply:
Is it the `export` that you don't like? Prefer it at the end?